### PR TITLE
lightning: refine dsecription of check-requirements

### PR DESCRIPTION
**Pull request by hfxsd**
### What is changed, added or deleted? (Required)

Refine dsecription of check-requirements

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [ ] v7.3 (TiDB 7.3 versions)
- [x] v7.2 (TiDB 7.2 versions)
- [x] v7.1 (TiDB 7.1 versions)
- [x] v7.0 (TiDB 7.0 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [x] v5.4 (TiDB 5.4 versions)
- [x] v5.3 (TiDB 5.3 versions)
- [x] v5.2 (TiDB 5.2 versions)
- [x] v5.1 (TiDB 5.1 versions)
- [x] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:https://github.com/pingcap/docs-cn/pull/14507
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-lightning/tidb-lightning-configuration.md [hunk 1]
```diff
@@ -42,7 +42,7 @@ max-backups = 14
 ### tidb-lightning task configuration
 
 [lightning]
-# Checks whether the cluster satisfies the minimum requirement before starting.
+# Checks whether the cluster satisfies the minimum requirement before starting, and check whether TiKV has more than 10% free space left after startup.
```
Comment (hfxsd) [inline tidb-lightning/tidb-lightning-configuration.md:45]:
```suggestion
# Checks whether the cluster satisfies the minimum requirement before starting the task,, and check whether TiKV has more than 10% free space left after startup.
```

File: tidb-lightning/tidb-lightning-configuration.md [hunk 2]
```diff
@@ -42,7 +42,7 @@ max-backups = 14
 ### tidb-lightning task configuration
 
 [lightning]
-# Checks whether the cluster satisfies the minimum requirement before starting.
+# Checks whether the cluster satisfies the minimum requirement before starting the task, and check whether TiKV has more than 10% free space left after startup.
```
Comment (hfxsd) [inline tidb-lightning/tidb-lightning-configuration.md:45]:
```suggestion
# Checks whether the cluster satisfies the minimum requirement before starting the task, and check whether TiKV has more than 10% free space left during running time.
```

File: tidb-lightning/tidb-lightning-configuration.md [hunk 3]
```diff
@@ -458,7 +458,7 @@ log-progress = "5m"
 | --enable-checkpoint *bool* | Whether to enable checkpoints (default = true) | `checkpoint.enable` |
 | --analyze *level* | Analyze tables after importing. Available values are "required", "optional" (default value), and "off" | `post-restore.analyze` |
 | --checksum *level* | Compare checksum after importing. Available values are "required" (default value), "optional", and "off" | `post-restore.checksum` |
-| --check-requirements *bool* | Check cluster version compatibility before starting (default = true) | `lightning.check-requirements` |
+| --check-requirements *bool* | Check cluster version compatibility before starting the task, and check whether TiKV has more than 10% free space left after startup. (default = true) | `lightning.check-requirements` |
```
Comment (hfxsd) [inline tidb-lightning/tidb-lightning-configuration.md:461]:
```suggestion
| --check-requirements *bool* | Check cluster version compatibility before starting the task, and check whether TiKV has more than 10% free space left during running time. (default = true) | `lightning.check-requirements` |
```

**Review by D3Hunter — APPROVED** [review]

**Review by Frank945946 — APPROVED** [review]

**Review by lilin90 — APPROVED** [review]
/lgtm

**Pull request merged**